### ISSUE TITLE
Smart contract panicked: Callback computation 0 was not successful

### ISSUE THREAD

**Issue by volovyks** [near/mpc#870]
https://nearblocks.io/txns/GJViX5rAWhWLCZ2fn29EWMWaCuBTgbLzjXhxn6pNUDHH#

**Comment by ppca** [comment]
another case here: https://testnet.nearblocks.io/txns/EsZArP4ihG3FCb2a3uYM6CcC9WkQsDKUctJ6JTReyGUH#execution

**Comment by ppca** [comment]
https://testnet.nearblocks.io/txns/D3BopvHBmpL1KHJPEfwmadLvqWf2fSXUfh2DZSAeRJRq
ok I was able to reproduce the problem. 
Issue was the prepaid gas we set for the clear_state_on_finish was not enough. 
As we can see in the 2 examples that failed, they consumed 7+ Tgas as compared to normal 3+ Tgas for clear_state_on_finish. 
When I reproduce this, I set the prepaid gas for clear_state_on_finish to be low (2 Tgas), then I see I can constantly get the `"Smart contract panicked: Callback computation 0 was not successful"` error.
Will double prepaid gas for clear_state_on_finish and return_signature_on_finish.
